code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1982,6 +1982,212 @@
+	result = "screamingviking"
+	required_reagents = list("vodka" = 1, "vermouth" = 1, "limejuice" = 1, "rum" = 1, "gin" = 1, "tonic" = 1)
+	result_amount = 6
+
+/datum/chemical_reaction/drinks/vilelemon
+	name = "Vile Lemon"
+	id = "vilelemon"
+	result = "vilelemon"
+	required_reagents = list("lemonade" = 5, "spacemountainwind" = 1)
+	result_amount = 6
+
+/datum/chemical_reaction/drinks/dreamcream
+	name = "Dream Cream"
+	id = "dreamcream"
+	result = "dreamcream"
+	required_reagents = list("milk" = 2, "cream" = 1, "honey" = 1)
+	result_amount = 4
+
+/datum/chemical_reaction/drinks/robustin
+	name = "Robustin"
+	id = "robustin"
+	result = "robustin"
+	required_reagents = list("antifreeze" = 1, "phoron" = 1, "fuel" = 1, "vodka" = 1)
+	result_amount = 4
+
+/datum/chemical_reaction/drinks/virginsip
+	name = "Virgin Sip"
+	id = "virginsip"
+	result = "virginsip"
+	required_reagents = list("driestmartini" = 1, "water" = 1)
+	result_amount = 2
+
+/datum/chemical_reaction/drinks/chocoshake
+	name = "Chocolate Milkshake"
+	id = "chocoshake"
+	result = "chocoshake"
+	required_reagents = list("milkshake" = 1, "coco" = 1)
+	result_amount = 2
+
+/datum/chemical_reaction/drinks/berryshake
+	name = "Berry Milkshake"
+	id = "berryshake"
+	result = "berryshake"
+	required_reagents = list("milkshake" = 1, "berryjuice" = 1)
+	result_amount = 2
+
+/datum/chemical_reaction/drinks/coffeeshake
+	name = "Coffee Milkshake"
+	id = "coffeeshake"
+	result = "coffeeshake"
+	required_reagents = list("milkshake" = 1, "coffee" = 1)
+	result_amount = 2
+
+/datum/chemical_reaction/drinks/jellyshot
+	name = "Jelly Shot"
+	id = "jellyshot"
+	result = "jellyshot"
+	required_reagents = list("cherryjelly" = 4, "vodka" = 1)
+	result_amount = 5
+
+/datum/chemical_reaction/drinks/slimeshot
+	name = "Named Bullet"
+	id = "slimeshot"
+	result = "slimeshot"
+	required_reagents = list("slimejelly" = 4, "vodka" = 1)
+	result_amount = 5
+
+/datum/chemical_reaction/drinks/negroni
+	name = "Negroni"
+	id = "negroni"
+	result = "negroni"
+	required_reagents = list("gin" = 1, "bitters" = 1, "vermouth" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/drinks/cloverclub
+	name = "Clover Club"
+	id = "cloverclub"
+	result = "cloverclub"
+	required_reagents = list("berryjuice" = 1, "lemonjuice" = 1, "gin" = 3)
+	result_amount = 5
+
+/datum/chemical_reaction/drinks/oldfashioned
+	name = "Old Fashioned"
+	id = "oldfashioned"
+	result = "oldfashioned"
+	required_reagents = list("whiskey" = 3, "bitters" = 1, "sugar" = 1)
+	result_amount = 5
+
+/datum/chemical_reaction/drinks/whiskeysour
+	name = "Whiskey Sour"
+	id = "whiskeysour"
+	result = "whiskeysour"
+	required_reagents = list("whiskey" = 2, "lemonjuice" = 1, "sugar" = 1)
+	result_amount = 4
+
+/datum/chemical_reaction/drinks/daiquiri
+	name = "Daiquiri"
+	id = "daiquiri"
+	result = "daiquiri"
+	required_reagents = list("rum" = 3, "limejuice" = 2, "sugar" = 1)
+	result_amount = 5
+
+/datum/chemical_reaction/drinks/mintjulep

```

Comment:
You added a reaction for a reagent that doesn't seem to exist.

Reply:
I'm having the devil's time figuring this, what reagent out of these is the nonexistent one? Same for both of these.

Reply:
Sorry for the delayed response, didn't see this. We don't have "lovepotion" or "mintjulep" reagents.

Reply:
Ah, I see. Forgot to define them in the actual chemicals list. Made the recipe without actually making the drink. I'll fix that tonight.